### PR TITLE
fix(Hero): update alert text color

### PR DESCRIPTION
**Pull request by BRKalow**
🎟️ [Asana Task]()
🔍 [Preview Link](https://react-components-git-{branch-slug}-hashicorp.vercel.app)

---

## Description

Make the nested `Alert` in `Hero` have the opposite text color of what the `backgroundTheme` prop is specified as.

Screen Shot 2021-09-29 at 11 01 15 AM

### PR Checklist 🚀

Items in this checklist may not may not apply to your PR, but please consider each item carefully.

- [ ] Add Asana and Preview links above.
- [ ] Conduct thorough self-review.
- [ ] Add or update tests as appropriate.
- [ ] Conduct reasonable cross browser testing for both compatibility and responsive behavior (We have a [Sauce Labs](https://app.saucelabs.com/) account for this, if you don't have access, just ask!).
- [ ] Conduct reasonable accessibility review (use the [WAS](https://accessible.org/Web-Accessibility-Standards-WAS-2.pdf) as a guide or an [axe browser plugin](https://www.deque.com/axe/) until we establish more formal checks).
- [ ] Identify (in the description above) and document (add Asana tasks on [this board](https://app.asana.com/0/1100423001970639/list)) any technical debt that you're aware of, but are not addressing as part of this PR.

### CODE REVIEW

File: packages/hero/index.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ function Hero({ data, centered, gaPrefix, className, videoControlsTop }) {
               url={alert.url}
               tag={alert.tag}
               text={alert.text}
-              textColor={backgroundTheme}
+              textColor={backgroundTheme === 'light' ? 'dark' : 'light'}
```
Comment (thiskevinwang) [inline packages/hero/index.js:68]: 2-second hand-wavy idea for the future

💭  We could pass `"invert"` that denotes `backgroundTheme === 'light' ? 'dark' : 'light'` logic
- basically, _the opposite of the current theme_

Comment (BRKalow) [reply packages/hero/index.js:68]: Pass `invert` to `Alert`? I wonder how we would implement that, as that would require `Alert` to be aware of its parent background color 🤔

Comment (thiskevinwang) [reply packages/hero/index.js:68]: ^ inspired by how Vercel handles this at the global level w/ `.invert-theme`
image

**Review by thiskevinwang — APPROVED** [review]

File: packages/hero/index.js (hunk 1 above)
Comment (BRKalow) [reply packages/hero/index.js:68]: Gotcha. Yeah I guess we'd need a notion of a proper theme abstraction which we could then "invert." 😅 

Good idea!

Comment (thiskevinwang) [reply packages/hero/index.js:68]: Probably a farfetched thought — (I'm missing some pieces of how theming is handled)

**Review by jescalan — APPROVED** [review]
Thanks for cleaning this up - noticed this bug the other day and it drove me nuts

**Pull request merged**